ui/lib/core/addon/components/form-field.hbs

```diff
@@ -40,7 +40,12 @@
                 class="has-left-margin-xs has-text-black is-size-7"
                 data-test-radio-label={{or val.label val.value val}}
               >
-                <span>{{or val.label val.value val}}</span>
+                {{or val.label val.value val}}
+                {{#if val.helpText}}
+                  <Hds::TooltipButton @text={{val.helpText}} aria-label="More information">
```

Comment:
I used Hds here because `InfoTooltip` styling was too low and didn't have the patience to CSS

Reply:
<img width="551" alt="Screenshot 2024-02-16 at 8 17 23 PM" src="https://github.com/hashicorp/vault/assets/68122737/d24fb916-faa7-49d2-939c-6eceffc5c7fb">